### PR TITLE
Unify yarn subcommand naming

### PR DESCRIPTION
**Pull request by nkrkv**
The PR slightly improves DX making the yarn commands more uniform. Now, there’s a clear separation of axes: verb (what to do), colon, subject (where to do). Subjects now have shorter names and imply `xod-`, `xod-client-` prefixes:

Verbs:

- `build`
- `dev`
- `dist`
- `start`

Subjects:

- `electron`
- `browser`
- `cli`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,34 +52,36 @@ Repository commands
...
-- `yarn build` — build, transpile, pack
-- `yarn dev-browser` — run dev-version of browser IDE on localhost
-- `yarn electron-dist` — build OS-specific distributive
+- `yarn build` — build, transpile, pack all
+- `yarn build:electron` — build desktop IDE only
+- `yarn build:cli` — build CLI tools only
+- `yarn dev:browser` — run dev-version of browser IDE on localhost
+- `yarn dist:electron` — build OS-specific distributive of desktop IDE
 - `yarn test` — run unit tests
 - `yarn test:watch` — like test, but watches for changes with auto-retest
```
Comment (evgenykochetkov) [inline README.md]: `test:watch` does not conform to new naming scheme

Comment (nkrkv) [reply README.md]: Good catch. Do you have a suggestion? `yarn test-watch`?

Comment (evgenykochetkov) [reply README.md]: Sounds good 👍🏻
Or we could just get rid of it. It's set up only for `xod-project` and `xod-client` anyway

Comment (nkrkv) [reply README.md]: OK. The question is do we use it? I’m not. @evgenykochetkov @brusherru do you have the development pattern with tests running on file change?

Comment (evgenykochetkov) [reply README.md]: I'm not using it.

Comment (brusherru) [reply README.md]: I'm not using it too :)

Comment (nkrkv) [reply README.md]: OK, removed it.

**Pull request merged**